apps/cookbook/src/app/showcase/modal-showcase/modal-showcase.component.html

```diff
@@ -219,67 +219,116 @@ <h3 #drawerSupplementaryAction>Drawer - supplementary action</h3>
   function which is invoked when the supplementary button is selected.
 </p>
 <cookbook-example-viewer [ts]="advancedConfigExample.drawerCodeSnippet"></cookbook-example-viewer>
-<h2 #routeBasedBasic>Route based modals</h2>
+
+<h2 #routeBasedBasic>Route-based modals</h2>
 
 <p>
-  Instead of providing the modal with a component, you can configure a route to be opened in a
-  modal. This is done by a secondary outlet named
+  Routed modals are useful for creating wizard-type flows where the same modal is reused for showing
+  multiple individual components one at a time. Modal routes will automatically open up a modal if
+  none is already opened which also means they are ideal for deep-linking directly to routes of
+  components that should be presented in a modal.
+</p>
+
+<cookbook-iphone
+  class="large-example"
+  src="/examples/modal-route"
+  viewMode="full-size"
+  [showViewModeToggle]="true"
+  showExternalLink="true"
+></cookbook-iphone>
+
+<cookbook-example-viewer
+  [html]="outletExample.template"
+  [ts]="outletExample.defaultCodeSnippet"
+></cookbook-example-viewer>
+
+<p>
+  Showing a component on its associated route inside a modal is achieved by configuring the route
+  with a secondary outlet named
```

Comment:
Consider linking to Angular docs:
```suggestion
  with a <a href="https://angular.io/api/router/Route#outlet" target="_blank"
    class="kirby-external-icon">secondary outlet</a> named
```